### PR TITLE
feat: banner conference (#84)

### PR DESCRIPTION
**Pull request by iflashlord**
Component has optional view as **parentheses**, **curly**, **greater** and **none**
Responsive applied by one media query
I have also tried to develop the component which I have found a problem regarding attaching the CSS files regarding the Shortcodes. (I have done a trick that is not really efficient but works).

### CODE REVIEW

File: src/_components/shortcodes/banner/banner.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+<g id="Button_play" clip-path="url(#clip-Button_play)">
+  <rect width="72" height="72" fill="transparent"/>
+  <g id="Group_503" data-name="Group 503" transform="translate(-0.204)">
+    <circle id="Ellipse_4" data-name="Ellipse 4" cx="36" cy="36" r="36" transform="translate(0.204)" fill="#fe796a"/>
+    <path id="Path_120" data-name="Path 120" d="M35.213,0V13.347L17.606,25.291,0,13.347V0Z" transform="translate(26.874 53.282) rotate(-90)"/>
+  </g>
+</g>
+</svg>
+`;
+const style = `
```
Comment (edwinm) [inline src/_components/shortcodes/banner/banner.js:16]: Is it possible to store the css in a .css file?

**Review by edwinm — CHANGES_REQUESTED** [review]
Looks good! I've one comment/request, though.

**Comment by iflashlord** [comment]
I have been checking that as well; I have put CSS in the file, but it did not attach to the project by using short-code in pages; I checked the configuration and spent some time to find a way to change the config, but I did not find a way to solve that. You can check other shortcodes as well. There are no styles attached to the main CSS after serving.
I even put the same CSS in the file next to the component in my PR.
I would say it stems from a lack of configuration for attaching CSS.

File: src/_components/shortcodes/banner/banner.js (hunk 1 above)
Comment (iflashlord) [reply src/_components/shortcodes/banner/banner.js:16]: I have been checking that as well; I have put CSS in the file, but it did not attach to the project by using short-code in pages; I checked the configuration and spent some time to find a way to change the config, but I did not find a way to solve that. You can check other shortcodes as well. There are no styles attached to the main CSS after serving.
I even put the same CSS in the file next to the component in my PR.
I would say it stems from a lack of configuration for attaching CSS.

**Pull request merged**